doc/13-repack.md

```diff
@@ -0,0 +1,30 @@
+# Repacking your application in a new phar
+
+You have created a castor application, with many tasks, and you want to
+distribute it as a single phar file? Castor can help you with that.
+
+In your project, install castor as a dependency:
+
+```bash
+composer require castor/castor
+```
+
+You'll also need
+[box](https://github.com/box-project/box/blob/main/doc/installation.md#installation)
+to create the phar.
+
+Then, run the repack command to create the new phar:
+
+```
+vendor/bin/castor repack
+```
```

Comment:
Before that we should add a note about `phar.readonly` setting in php.ini so people are not lost (see https://www.php.net/manual/en/phar.configuration.php for more details)